server/controllers/UserController.js

```diff
@@ -37,41 +38,56 @@ exports.sendMessage = async (parent, args, context) => {
     let room = await Room.findOne({
       _id: args.roomId,
       members: { $in: [context.currentUser.id] },
-    });
+    }).populate("members");
```

Comment:
i'm populating the members so i can check if mentioned usernames in the message body are members of the current room